### ISSUE TITLE
3q

### ISSUE THREAD

**Issue by NakedGravity** [jvavrek/NIMB2018#4]
1) Does your code take into account the probability of a coherent process (so called, "pure" NRF)?
2) If so, what is the error in determining the probability of NRF? More or less than 30%?
3) And finally, can I use G4NRF for Lu?  Or maybe user can add their own isotopes in your database.

P.S. I can't send you an email to the address you provided.

**Comment by jvavrek** [comment]
> 1. Does your code take into account the probability of a coherent process (so called, "pure" NRF)?

I'm unfamiliar with the "coherent" distinction you raise, but the code will take into account the probabilities for so-called "elastic" vs "inelastic" NRF, i.e., (ground state -> excited state -> ground state) vs (ground state -> excited state -> intermediate state). Another coherent process, Rayleigh scattering, is not part of G4NRF itself but is part of the regular EM Geant4 libraries (but might be disabled by default, can't remember).

>  2. If so, what is the error in determining the probability of NRF? More or less than 30%?

The probabilities involved depend on the databases used and how accurate they are, as well as how accurately your Geant4 model reflects reality. [In this paper](https://arxiv.org/abs/1807.02596) we found agreement between Geant4+G4NRF _absolute_ NRF count rates to the level of 15-20% in U-238 and 8% in Al-27. So with careful modeling and well-validated resonance parameters, <30% error to data is certainly achievable. We also compared Geant4+G4NRF to some (semi-)analytical modeling and found 1-3% agreement [here](https://arxiv.org/abs/1807.01701).

>  3. And finally, can I use G4NRF for Lu?  Or maybe user can add their own isotopes in your database.

Lu should certainly be possible. If you look at the existing database, there are entries for most Lu isotopes between A=151 and A=180, including the natural 175 and 176. I haven't worked with Lu myself, though, so I can't comment on how accurate or complete those database entries are. The database in general is based off of ENSDF, so check [this page](https://www.nndc.bnl.gov/nudat3/getdatasetClassic.jsp?nucleus=175LU&unc=nds) and look for reference H to see what is likely available for Lu-175.

Adding additional NRF lines or new nuclides entirely is a bit of a pain, as you would need to edit three files in the database per nuclide:
- zZZ.aAAA
- level_table_AAA.dat
- gamma_table_AAA.dat

But hopefully the existing databases are sufficient. Hope this helps!
